config/realia-strict.conf

```diff
@@ -285,6 +285,7 @@ self-call-recursive:			skip
 record-contains-depending-clause:	unconformable
 defaultbyte:				0		# not verified, but likely like IBM
 picture-l:				unconformable
+odo-in-redefines:			unconformable # to be checked
```

Comment:
Haven't found anything about it in its COBOL Reference Guide [that's in windows HLP format...] it has some rules on `OCCURS` but doesn't specify anything between that and `REDEFINES`.

In any case that would be `# not verified, but likely like IBM`